compiler/plugins/target/AMD-AIE/aie/AMDAIECreatePathFindFlows.cpp

```diff
@@ -1359,40 +568,38 @@ bool AMDAIEPathfinderPass::reconnectConnectOps(const OpBuilder &builder,
                                                SwitchboxOp sw,
                                                ConnectOp problemConnect,
                                                bool isIncomingToSW,
-                                               WireBundle problemBundle,
+                                               StrmSwPortType problemBundle,
                                                int problemChan, int emptyChan) {
   bool hasEmptyChannelSlot = true;
   bool foundCandidateForFixup = false;
   ConnectOp candidate;
-  if (isIncomingToSW) {
+  if (isIncomingToSW)
     for (ConnectOp connect : sw.getOps<ConnectOp>()) {
       if (connect.getDestBundle() == problemBundle &&
           connect.getDestChannel() == problemChan) {
         candidate = connect;
         foundCandidateForFixup = true;
       }
       if (connect.getDestBundle() == problemBundle &&
-          connect.getDestChannel() == emptyChan) {
+          connect.getDestChannel() == emptyChan)
         hasEmptyChannelSlot = false;
-      }
     }
-  } else {
+  else
     for (ConnectOp connect : sw.getOps<ConnectOp>()) {
       if (connect.getSourceBundle() == problemBundle &&
           connect.getSourceChannel() == problemChan) {
         candidate = connect;
         foundCandidateForFixup = true;
       }
       if (connect.getSourceBundle() == problemBundle &&
-          connect.getSourceChannel() == emptyChan) {
+          connect.getSourceChannel() == emptyChan)
         hasEmptyChannelSlot = false;
-      }
     }
```

Comment:
Style guide says to keep braces here: https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching